dependencies.list

```diff
@@ -1,4 +1,4 @@
-VERSION=3.18.0
-REALM_CORE_VERSION=5.23.4
-REALM_SYNC_VERSION=4.7.5-catalyst.3
+VERSION=3.18.1
+REALM_CORE_VERSION=5.23.5
+REALM_SYNC_VERSION=4.7.8-catalyst.3
```

Comment:
This is not a release that exists.

Reply:
Are the catalyst releases done manually for now?

Reply:
4.7.5-catalyst was 4.7.5 rebuilt with the build system changes needed to support catalyst. Those have now been merged and the normal 4.7.8 release supports it.